api/fastapi_service/main.py

```
@@ -5,46 +5,54 @@
-@app.on_event("startup")
-async def startup():
-    await database.connect()
-    services.init_db(cities_info=cities_info)
-
-@app.on_event("shutdown")
-async def shutdown():
-    await database.disconnect()
 
 if __name__ == "__main__":
     metadata.create_all(engine)
     run("main:app", host="0.0.0.0", port=getenv("PORT", 8901), reload=True)
```

Ruff (0.14.1)

Possible binding to all interfaces

(S104)